styleguide/forms.py

```diff
@@ -0,0 +1,39 @@
+from django import forms
+
+class ExampleForm(forms.Form):
```

Comment:
If there are multiple example forms, lets give this a less general name. I vote for `AllThatAndTheKitchenSinkForm`

Reply:
I would also accept "Everything Bagel"